server/graphql/schema/ingestion/resolvers/mutations/ingestData.ts

```diff
@@ -288,20 +288,57 @@ class IngestDataWorker extends ResolverBase {
     }
 
     private async validateIdentifier(identifier: IngestIdentifierInput): Promise<IdentifierResults> {
-        // compute identifier; for ARKs, extract the ID from a URL that may be housing the ARK ID
+
+        const regexArk : RegExp = new RegExp("ark:\/65665\/[0-9a-zA-Z-]*([^@!%$&#*+=.,\"\'\^]+$)");
```

Comment:
Good advice. Thanks. Since other identifier types don't have collection implementations (yet) do see a better place for validating those inputs (simple regex) prior to implementing the full interfaces?